### ISSUE TITLE
Regarding Problems with Editing and Syncing Lyrics in the Lyric Editor

### ISSUE THREAD

**Issue by daoxi** [jacquesh/foo_openlyrics#123]
In Lyric Editor, when I click on "**Sync line using 'offset' tag**" button (which is supposed to "**add an 'offset' tag that synchronizes all lines**" according to the hover-on description), I get an error message saying "**The currently-selected line does not have a timestamp**", I feel like this is probably not intended? (because the offset tag does not depend on the user to select a line, it's global for all lines)

Also, when editing LRC lyrics, I found that for lines with only a timestamp and empty lyrics, for example (2nd line):
```
[00:12.00]Line 1 lyrics
[00:17.20]
[00:21.10]Line 3 lyrics
```
There is somehow an extra space after the timestamp when that space doesn't exist in the original lyrics (for example: "[00:17.20]" becomes "[00:17.20] ")

**Comment by jacquesh** [comment]
Dupe of #107, which is fixed for the next release already.

I know about the additional space, I added that intentionally although off the top of my head right now I don't recall what the reason was. It won't get added unless you make edits, is it causing a problem?

**Comment by daoxi** [comment]
> Dupe of https://github.com/jacquesh/foo_openlyrics/issues/107, which is fixed for the next release already.

Good to know, thanks for fixing.

> I know about the additional space, I added that intentionally although off the top of my head right now I don't recall what the reason was.

Maybe the previous reason no longer applys now (e.g. a past limitation that has now been fixed, such as wasn't able to display empty lyrics so you had to add an extra space as temporary workaround or something), perhaps that could be why you don't remember it anymore?

>  It won't get added unless you make edits, is it causing a problem?

The problem is that I wish to keep the rest of the lyrics **intact** when I just wanna do some simple editing, and adding extra space after the timestamp will break the format consistency of empty lyrics. For example, if I want to do some batch regex operation to find all empty lyrics lines by looking for an **immediate line ending** (e.g. CRLF) after the timestamp, now that will **stop working** due to the extra space.

**Comment by jacquesh** [comment]
Ok I checked the code, the space was getting added so the the "empty line" selection is visible in unsynced lyrics when using the "Synchronise selected line" button. The extra space *should* already be getting removed on save (its only there in OpenLyrics' editor). Regardless I've changed it to not add the space when there's a timestamp (since the timestamp serves the same purpose of providing something to visibly highlight there).
